.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,75 @@
+---
+name: Bug report
+description: Report a bug or an issue you've found with Trino Python client
+labels: bug
+body:
+  - type: textarea
+    attributes:
+      label: Expected behavior
+      description: What do you think should have happened
+      placeholder: >
+        A clear and concise description of what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Actual behavior
+      description: Describe what actually happened
+      placeholder: >
+        A clear and concise description of what actually happened.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps To Reproduce
+      description: This will help us reproduce your issue
+      placeholder: >
+        In as much detail as possible, please provide steps to reproduce the issue.
+        Sample code that triggers the issue, relevant server settings, etc is all very helpful here.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Screenshots and log output
```

Comment:
I don't think suggesting screenshots is a good idea. It would be better to suggest only logs so that we can copy-and-paste for debugging. I agree if this is GUI application though. What do you think about it?